src/main/java/org/folio/dew/batch/acquisitions/edifact/CompositePOConverter.java

```diff
@@ -28,7 +28,7 @@ public void convertPOtoEdifact(EDIStreamWriter writer, CompositePurchaseOrder co
     int messageSegmentCount = 0;
 
     messageSegmentCount++;
-    writePOHeader(ediFileConfig.getFileId(), writer);
+    writePOHeader(compPO, writer);
```

Comment:
I am out of context, but is it suppose to be comPO.getPoNumber() because previously it was ediFileConfig.getFileId()